reexport_zip.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
```

Comment:
If we need this shell snippet then move to ./scripts/
Alternatively the snippet could be absorbed in to the entrypoint and deprecated.